zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/AbstractInterpreter.java

```diff
@@ -47,6 +51,35 @@ public InterpreterResult interpret(String st,
     return internalInterpret(st, context);
   }
 
+  static String interpolate(String cmd, ResourcePool resourcePool) {
+    Pattern zVariablePattern = Pattern.compile("([^{}]*)([{]+[^{}]*[}]+)(.*)", Pattern.DOTALL);
```

Comment:
Move it outside as static variable ?

Reply:
With `static` and `final` :grin: 
https://rules.sonarsource.com/java/tag/performance/RSPEC-4248

Reply:
I fixed. But No difference was found from the execution time of `ZeppCtxtVariableTest`.
Below is the execution time of 18 tests of `ZeppCtxtVariableTest`. (measured 10 times each)
statistics | before (ms) | after (ms) 
-------|--------|-------
average | 235.6| 234.6
std. dev. | 12.6 | 26.5